google/cloud/spanner/integration_tests/data_types_integration_test.cc

```diff
@@ -292,6 +305,30 @@ TEST_F(PgDataTypeIntegrationTest, WriteReadNumeric) {
   EXPECT_THAT(result, IsOkAndHolds(UnorderedElementsAreArray(data)));
 }
 
+TEST_F(DataTypeIntegrationTest, WriteReadProtoEnum) {
+  if (UsingEmulator()) GTEST_SKIP() << "emulator does not support PROTO";
+
+  std::vector<ProtoEnum<testing::Genre>> const data = {
+      testing::Genre::POP,
```

Comment:
Oh noes... An enum? 😝 

Where is PROGRESSIVE_JAZZ, and BEPOP, and ITALIAN_BARROQUE?  And let's hope there are no macros called `ROCK` in any OS we need to support.

Reply:
BEPOP!